uv.lock

```diff
@@ -606,9 +642,10 @@ wheels = [
 
 [[package]]
 name = "craft-parts"
-version = "2.18.0"
```

Comment:
I believe this is what's causing the breakage?

Reply:
https://github.com/canonical/craft-parts/issues/1207

https://github.com/canonical/craft-parts/pull/1208